src/CylindricalDipoleMagnet_geo.cpp

```diff
@@ -43,19 +43,22 @@ static Ref_t build_magnet(Detector& dtor, xml_h e, SensitiveDetector /* sens */)
   const string module_name = "Quad_magnet";
 
   const string yoke_vis = dd4hep::getAttrOrDefault<std::string>(x_det, _Unicode(vis), "FFMagnetVis");
-  const string coil_vis = dd4hep::getAttrOrDefault<std::string>(coil, _Unicode(vis), "FFMagnetCoilVis");
+  //const string coil_vis = dd4hep::getAttrOrDefault<std::string>(coil, _Unicode(vis), "FFMagnetCoilVis");
 
   sdet.setAttributes(dtor, assembly, x_det.regionStr(), x_det.limitsStr(), yoke_vis);
 
   // -- yoke
-  Tube   yoke_tube(app_r + coil_y, dim_r, 0.5 * dim_z);
+  //Tube   yoke_tube(app_r + coil_y, dim_r, 0.5 * dim_z);
+  Tube   yoke_tube(app_r, dim_r, 0.5 * dim_z);
   Volume yoke_vol("yoke_vol", yoke_tube, iron);
   auto   yoke_pv = assembly.placeVolume(yoke_vol);
   yoke_pv.addPhysVolID("element", 1);
   DetElement yoke_de(sdet, "yoke_de", 1);
   yoke_de.setPlacement(yoke_pv);
   yoke_de.setAttributes(dtor, yoke_vol, x_det.regionStr(), x_det.limitsStr(), yoke_vis);
 
+  /*
```

Comment:
Remove commented lines or explain in comment why should be retained.

Reply:
Done.